.github/workflows/ci.yml

```diff
@@ -0,0 +1,150 @@
+name: CI
+on:
+  pull_request:
+    branches: [development]
+    paths-ignore:
+      - '*.md'
+  push:
+    branches: [development]
+    paths-ignore:
+      - '*.md'
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+jobs:
+  test-build-typescript:
+    name: Build Typescript
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node and npm registry
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          registry-url: 'https://registry.npmjs.org/'
+          cache: 'yarn'
+      - name: Install node_modules
+        run: yarn install --frozen-lockfile
+      - name: Install node_modules (example/)
+        run: yarn install --frozen-lockfile --cwd example
+      - name: Compile TypeScript
+        run: yarn typescript
+  test-build-android:
+    name: Build Android
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node and npm registry
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          registry-url: 'https://registry.npmjs.org/'
+          cache: 'yarn'
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Install node_modules
+        run: yarn install --frozen-lockfile
+      - name: Install node_modules (example/)
+        run: yarn install --frozen-lockfile --cwd example
+      - name: Set up Gradle cache
+        uses: gradle/gradle-build-action@v2
+        with:
+          cache-read-only: ${{ github.ref != 'refs/heads/development' }}
+      - name: Build Android example
+        run: ./gradlew assembleDebug --build-cache
+        working-directory: example/android
+  test-build-ios:
+    name: Build iOS
+    runs-on: macOS-latest
```

Comment:
For the tvOS build, we used an specific `macOS` version. Is there a reason, why we want to use the latest here, but a specific version for tvOS?

Reply:
@matamegger That was a mistake, fixed in 97c3d19